packages/Statie/packages/HeadlineAnchorLinker/src/EventSubscriber/GeneratorFilesHeadlineAnchorLinkerEventSubscriber.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\Statie\HeadlineAnchorLinker\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symplify\Statie\Event\BeforeRenderEvent;
+use Symplify\Statie\Generator\Configuration\GeneratorConfiguration;
+use Symplify\Statie\Generator\Renderable\File\AbstractGeneratorFile;
+use Symplify\Statie\HeadlineAnchorLinker\HeadlineAnchorLinker;
+
+final class GeneratorFilesHeadlineAnchorLinkerEventSubscriber implements EventSubscriberInterface
```

Comment:
Just curious - when and how is this event subscriber used?

Reply:
You can always find reference to that place in `getSubscribedEvents()`.

https://github.com/Symplify/Symplify/pull/1395/files/23473c4529248acf27e462589d26522535fba8ee#diff-d90f85371a1b1a9f18f908cd4648accdR36

![image](https://user-images.githubusercontent.com/924196/51798906-3aa69300-221a-11e9-8b59-2ba4e8fda57a.png)

Just look for `new BeforeRenderEvent` or `new <event-class>` to find the place.

Reply:
There you'll find place it's *invoked*. 

Here it's *used*.